### ISSUE TITLE
Update to Storybook 4.0

### ISSUE THREAD

**Issue by chengcyber** [dump247/storybook-state#10]
Thanks for this addon!
__
I am using storybook 4.0, and I found a bug, plz see the screenshot. The `state` info should be displayed in `STATE` panel, instead of `KNOBS` panel.

![image](https://user-images.githubusercontent.com/16147702/47290410-ee9fed00-d631-11e8-9cdf-0739a1129f2b.png)

It seems to me storybook 4 updates the register logic, which causes this bug...

my dependencies

```
    "@dump247/storybook-state": "^1.5.0",
    "@storybook/addon-actions": "^4.0.0-rc.0",
    "@storybook/addon-info": "^4.0.0-rc.0",
    "@storybook/addon-knobs": "^4.0.0-rc.0",
    "@storybook/addon-links": "^4.0.0-rc.0",
    "@storybook/addon-options": "^4.0.0-rc.0",
    "@storybook/react": "^4.0.0-rc.0",
```

**Comment by AugustinGrigorov** [comment]
Also having this issue. It is displayed in every other panel at the bottom.

**Comment by romeovs** [comment]
Same here

**Comment by dump247** [comment]
Should be resolved in v1.6.0
